src/_pytest/config/__init__.py

```diff
@@ -1130,8 +1130,8 @@ def getini(self, name: str):
     def _getini(self, name: str) -> Any:
         try:
             description, type, default = self._parser._inidict[name]
-        except KeyError:
-            raise ValueError("unknown configuration value: {!r}".format(name))
+        except KeyError as e:
+            raise ValueError("unknown configuration value: {!r}".format(name)) from e
```

Comment:
Here I think `from None` would be better.

```suggestion
        except KeyError:
            raise ValueError("unknown configuration value: {!r}".format(name)) from None
```